### DISCUSSION TITLE
Bun v1.0.6

### DISCUSSION THREAD

**Discussion by Jarred-Sumner** [oven-sh/bun#6444] [Announcements]
To install Bun v1.0.6:
```bash
curl https://bun.sh/install | bash
# or you can use npm
# npm install -g bun
```

To upgrade to Bun v1.0.6:
```bash
bun upgrade
```

**[Read Bun v1.0.6's release notes on Bun's blog](https://bun.sh/blog/bun-v1.0.6)**

## What's Changed
* Docs : Added instructions to run bun apps in daemon (PM2) to address … by @h2210316651 in https://github.com/oven-sh/bun/pull/5931
* fix #6416 by @dylan-conway in https://github.com/oven-sh/bun/pull/6430
* feat(install): support npm overrides/yarn resolutions, one level deep only by @paperdave in https://github.com/oven-sh/bun/pull/6435
* chore: add missing ending quote by @ImLunaHey in https://github.com/oven-sh/bun/pull/6436

## New Contributors
* @ImLunaHey made their first contribution in https://github.com/oven-sh/bun/pull/6436

**Full Changelog**: https://github.com/oven-sh/bun/compare/bun-v1.0.5...bun-v1.0.6

<hr /><em>This discussion was created from the release <a href='https://github.com/oven-sh/bun/releases/tag/bun-v1.0.6'>Bun v1.0.6</a>.</em>

**Comment by henshalb** [comment]
In the nuanced tapestry of existence, Bun's speed becomes a metaphor, eluding the constraints of possibility and inviting contemplation on the boundless realms of potentiality.

**Reply by n-ce** [reply]
**Simplified version** : Bun is so fast that it makes us think about how much is possible in the world.
